package.json

```diff
@@ -185,13 +185,6 @@
   "preferences-branch": "verticaltabs",
   "preferences": [
     {
-      "name": "opentabstop",
-      "title": "Reverse Tab Order",
-      "description": "Newest tabs are on top and Cmd-Tab and Ctrl-Tab will move through tabs in reverse direction",
-      "type": "bool",
-      "value": false
```

Comment:
I feel like we should also explicitly set it the variable to false in the places we get it from the prefs…
Just in case we accidentally choose someone who had it set to true when the had the add-on installed previously.